src/course-outline/configure-modal/ConfigureModal.test.jsx

```diff
@@ -0,0 +1,159 @@
+    initializeMockApp({
+      authenticatedUser: {
+        userId: 3,
+        username: 'abc123',
+        administrator: true,
+        roles: [],
+      },
+    });
+
+    store = initializeStore();
+    axiosMock = new MockAdapter(getAuthenticatedHttpClient());
+    useSelector.mockReturnValue(currentSectionMock);
+  });
+
+  it('renders ConfigureModal component correctly', () => {
+    const { getByText, getByRole } = renderComponent();
+    expect(getByText(`${currentSectionMock.displayName} Settings`)).toBeInTheDocument();
+    expect(getByText(messages.basicTabTitle.defaultMessage)).toBeInTheDocument();
+    expect(getByText(messages.visibilityTabTitle.defaultMessage)).toBeInTheDocument();
+    expect(getByText(messages.releaseDate.defaultMessage)).toBeInTheDocument();
+    expect(getByText(messages.releaseTimeUTC.defaultMessage)).toBeInTheDocument();
+    expect(getByRole('button', { name: messages.cancelButton.defaultMessage })).toBeInTheDocument();
+    expect(getByRole('button', { name: messages.saveButton.defaultMessage })).toBeInTheDocument();
+  });
+
+  it('switches to the Visibility tab and renders correctly', () => {
+    const { getByRole, getByText } = renderComponent();
+
+    const visibilityTab = getByRole('tab', { name: messages.visibilityTabTitle.defaultMessage });
+    fireEvent.click(visibilityTab);
+    expect(getByText(messages.sectionVisibility.defaultMessage)).toBeInTheDocument();
+    expect(getByText(messages.hideFromLearners.defaultMessage)).toBeInTheDocument();
+  });
+
+  it('disables the Save button and enables it if there is a change', () => {
+    const { getByRole, getByPlaceholderText, getByTestId } = renderComponent();
+
+    const saveButton = getByRole('button', { name: messages.saveButton.defaultMessage });
+    expect(saveButton).toBeDisabled();
+
+    const input = getByPlaceholderText('MM/DD/YYYY');
+    fireEvent.change(input, { target: { value: '12/15/2023' } });
+
+    const visibilityTab = getByRole('tab', { name: messages.visibilityTabTitle.defaultMessage });
+    fireEvent.click(visibilityTab);
+    const checkbox = getByTestId('visibility-checkbox');
+    fireEvent.click(checkbox);
+    expect(saveButton).not.toBeDisabled();
+  });
+
+  it('calls OnConfigureSubmit upon save', () => {
+    const { getByRole, getByPlaceholderText } = renderComponent();
+
+    const saveButton = getByRole('button', { name: messages.saveButton.defaultMessage });
+    const timeInput = getByPlaceholderText('HH:MM');
+
+    fireEvent.change(timeInput, { target: { value: '10:10' } });
+    fireEvent.click(saveButton);
+
+    expect(onConfigureSubmitMock).toHaveBeenCalledTimes(1);
+  });
+
+  it('calls the onClose function when the cancel button is clicked', () => {
+    const { getByRole } = renderComponent();
+
+    const cancelButton = getByRole('button', { name: messages.cancelButton.defaultMessage });
+    fireEvent.click(cancelButton);
+    expect(onCloseMock).toHaveBeenCalledTimes(1);
+  });
+});
```

Comment:
@CefBoud Both of these tests are not really testing anything, instead we can test `configureCourseSectionQuery` if removing these tests affect coverage.